src/cucumber/steps/blockchain/setActiveWallet.ts

```diff
@@ -38,3 +38,19 @@ export const setActiveWalletByIndexStep = async (index: number) => {
     `Active test wallet address set to address at index ${index}: ${testWallet}`,
   );
 };
+
+/**
+ * Fetches the wallet address by the provided index
+ * @param index- The index of the test wallet address to set as active.
+ * 
+ * @example
+ * import { fetchWalletByIndexStep } from "chukti";
+ * When("I fetch the wallet address at index {int}", fetchWalletByIndexStep);
+ */
+
+export const fetchWalletByIndexStep = async (index: number) => {
+  const testWallet = await getTestWalletAddress({ index });
+
```

Comment:
```suggestion

if (!world.chukti) {
    world.chukti = {};
}

world.chukti.lastResult = testWallet

```

We need to store it in the `lastResult` property so that user will be able to store this data using ``I store the result in {string} step definition

Reply:
Yes sir, I will be doing that and creating a new updated pull request! Thank you for inputs